packaging/graphql-engine/Dockerfile

```diff
@@ -9,4 +9,5 @@ COPY rootfs/srv/ /srv/
 
 HEALTHCHECK --start-period=10s CMD curl -f http://localhost:8080/healthz || exit 1
 
-CMD "${HGE_BINARY}" serve
+ENTRYPOINT [ "${HGE_BINARY}" ]
```

Comment:
Does this actually work? What interprets the environment variable?

Reply:
Right; after reading a bit closer I'm inclined to agree to not being expanded. If you know where the documentation / steps to reproduce the finished container is, point me in that direction. I built the ubuntu base container and pointed to it as a build arg but it wasn't enough since I don't know where `rootfs` comes from.

Does this have to be dynamic? I can create a shell entrypoint that traps the signal otherwise.

Reply:
If it's just the shell traps that you want, you can also run the Docker image with `--init` (or `init: true` in Docker Compose).

Reply:
yes, I just thought I'd contribute back so I can improve the state for everyone. Feel free to close if you think that's what's most appropriate at this stage.